Buy/Global.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  Global.swift
+//  Buy
+//
+//  Created by Dima Bart on 2017-02-24.
+//  Copyright © 2017 Shopify Inc. All rights reserved.
+//
+
+import Foundation
+
+private final class BundleCoordinator: NSObject {
+    
+    static func frameworkBundle() -> Bundle {
+        return Bundle(for: self.classForCoder())
```

Comment:
This is not really an appropriate use for `classForCoder()`. That is used for `NSCoding`, i.e. for encoding _instances_, not finding where a class was implemented.

Suggest you use `self.Type`.

What about making this a protocol extension on `NSObject`?